### DISCUSSION TITLE
Banner Size doesn't affect

### DISCUSSION THREAD

**Discussion by kyoz** [Poing-Studios/Godot-AdMob-Android-iOS#84] [Q&A]
Is any one tested to change Banner Size. I'v tried to change all the Size but it's all the same. Just a Mini Ad display as center bottom of the screen.

Info:

Godot: 3.3.2
Admob & Editor: Latest
Phone: Galaxy Note 10 Plus

**Comment by gumaciel** [comment]
Hi @kyoz , i'm testing here and it's working fine.

Ps: All the sizes will be "centered" on bottom or top

**Reply by kyoz** [reply]
Yay, the ads is display without problem. But due to the [docs](https://developers.google.com/admob/android/banner/smart). Smart Banner is display full width, if the ads not have enough width, it'll be fill with black background.

There is also [Adaptive Banner](https://developers.google.com/admob/android/banner/adaptive) which is very good we should consider add it to the plugin in the future.

**Reply by kyoz** [reply]
![image](https://user-images.githubusercontent.com/29815830/123024623-88cc1a80-d403-11eb-846c-9773a06f0058.png)

It's seem the plugin is only using BANNER size. So it have a fixed width and height

**Reply by gumaciel** [reply]
@kyoz issue created: https://github.com/Poing-Studios/Godot-AdMob-Android-iOS/issues/86 implementation of this new Banner Size will be done soon.
